src/partition/repart.c

```diff
@@ -1619,12 +1622,24 @@ static int context_dump_partitions(Context *context, const char *node) {
         Partition *p;
         int r;
 
-        t = table_new("type", "label", "uuid", "file", "node", "offset", "raw size", "size", "raw padding", "padding");
+        if (!arg_json && context->n_partitions == 0) {
+                puts("Empty partition table.");
```

Comment:
hmm, log_info() probably, i.e. if you redirect output to some file this shouldn't show up i am sure...